e2e/specs/detail.spec.ts

```diff
@@ -21,32 +22,11 @@ test.describe.parallel('Tour of Heroes - Detail page e2e tests', async () => {
     });
 
     const initialPages = ['Dashboard', 'Heroes'];
-    initialPages.forEach(initPage => {
-        test(`Save button click - Updates name and navigates user to previous page - ${initPage}`, async ({page}) => {
+    initialPages.forEach(startPageName => {
```

Comment:
I still don't think this test should be parametrized at all, but I think you got my idea.

Reply:
Yep. We had discussed. It's just example.